src/main/java/net/minecraftforge/forgespi/language/IModFileInfo.java

```diff
@@ -34,6 +34,7 @@ record LanguageSpec(String languageName, VersionRange acceptedVersions) {}
     List<LanguageSpec> requiredLanguageLoaders();
 
     boolean showAsResourcePack();
```

Comment:
If you make this a default implemented with the  expected value for current behaviour this can be pulled instantly without breaking binary compatibility

Reply:
Not a bad idea; given that the change it was made to go with in NeoForge is, itself, a breaking change, is this worth doing?

Reply:
It is probably worth it, because it means it is a straight merge instead of of having to make an entirely new major version

Reply:
Fair enough. I'll get that set up then